tests/unit/cgd-test.ttl

```diff
@@ -0,0 +1,137 @@
+@prefix ns1: <http://purl.org/oban/> .
```

Comment:
Test data is great, but it should be put in the ``tests/data`` directory. The next file seems rather large --- do we really need a 27K line file of test data?